tests/kafkatest/tests/core/downgrade_test.py

```diff
@@ -26,10 +27,12 @@
 from kafkatest.version import LATEST_0_9, LATEST_0_10, LATEST_0_10_0, LATEST_0_10_1, LATEST_0_10_2, LATEST_0_11_0, LATEST_1_0, LATEST_1_1, LATEST_2_0, LATEST_2_1, LATEST_2_2, LATEST_2_3, V_0_9_0_0, V_0_11_0_0, DEV_BRANCH, KafkaVersion
 
 class TestDowngrade(EndToEndTest):
+    PARTITIONS = 3
+    REPLICATION_FACTOR = 3
 
     TOPIC_CONFIG = {
-        "partitions": 3,
-        "replication-factor": 3,
+        "partitions": PARTITIONS,
```

Comment:
Is this necessary to be defined as constants?

Reply:
Yes, see https://github.com/apache/kafka/pull/8495/files#diff-11673271221c04c0861f9c5e074a9783R80